docs/gwcs/wcs_validation.rst

```diff
@@ -7,14 +7,18 @@ WCS validation
 The WCS is validated when an object is read in or written to a file.
 However, this happens transparently to the end user and knowing
 the details of the validation machinery is not necessary to use or
-construct a WCS object. 
+construct a WCS object.
 
 GWCS uses the `Advanced Scientific Data Format <https://asdf-standard.readthedocs.io/en/latest/>`__ (ASDF)
 to validate the transforms, coordinate frames and the overall WCS object structure.
```

Comment:
so something like this would be accurate enough?

```
... to serialize and deserialize GWCS objects (including transformations and frames) and to provide validation that the serialization is correct.
```

Reply:
Sure. Whomever phrased that is brilliant.

Reply:
^^^^^^